.github/workflows/sentinel-build.yml

```diff
@@ -0,0 +1,161 @@
+name: Sentinel Build (No Cache)
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "15 3 * * Sat" # 11:15PM EST every Friday
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  build-docker:
+  if: (github.repository_owner == 'wpilibsuite' && github.ref == 'refs/heads/main') || github.event_name != 'schedule'
```

Comment:
As with the other workflows, want to confine this to the allwpilib repo.

Reply:
I left this one as-is intentionally- I thought if there were a fork, you'd want this to run on it as well (assuming you were using the main branch)